src/gmt_support.c

```diff
@@ -3800,8 +3800,8 @@ GMT_LOCAL struct GMT_DATASET * gmtsupport_voronoi_shewchuk (struct GMT_CTRL *GMT
 			if (i >= n_int_edges) edge_use[i] = 1;	/* These edges will only be used once (sign is not important here) */
 		}
 		/* Need temp array to hold the coordinates of a single polygon [I hope this is large enough - don't know yet] */
-		xcoord = gmt_M_memory (GMT, NULL, n_int_edges/2, double);
-		ycoord = gmt_M_memory (GMT, NULL, n_int_edges/2, double);
+		xcoord = gmt_M_memory (GMT, NULL, n_int_edges, double);
```

Comment:
Doesn't this need to be 2*n_int_edges or some other value >n_int_edges for the crash to be resolved?